src/components/TsmlUI.tsx

```diff
@@ -241,40 +271,42 @@ export default function TsmlUI({
   }
 
   return (
-    <SettingsContext.Provider value={{ settings, strings }}>
-      <Global styles={globalCss} />
-      {state.loading ? (
-        <Loading />
-      ) : state.input.meeting && state.input.meeting in state.meetings ? (
-        <Meeting setState={setState} state={state} />
-      ) : (
-        <>
-          {settings.show.title && <Title state={state} />}
-          {settings.show.controls && (
-            <Controls state={state} setState={setState} />
-          )}
-          {(state.alert || state.error) && <Alert state={state} />}
-          {state.filtering ? (
-            <Loading />
-          ) : state.input.view === 'table' ? (
-            <Table
-              filteredSlugs={filteredSlugs}
-              inProgress={inProgress}
-              setState={setState}
-              state={state}
-            />
-          ) : (
-            <div style={{ display: 'flex', flexGrow: 1 }}>
-              <Map
+    <div style={{ minHeight: `calc(100dvh - ${occludedHeight}px)` }}>
```

Comment:
the diff looks a little weird below - the only change here is wrapping everything in this div ☝️

Reply:
LGTM, works on my machine